requirements/2.7/requirements.txt

```diff
@@ -40,9 +40,6 @@ django-tagging==0.3.1
 pytz==2013.8
 
 # Cthulhu
-M2crypto==0.22.3
```

Comment:
by-hand would be broken, that Is fair. I should be able to fix that.

re: aside: I agree. Working with upstream packages is the goal. devmode working on fedora is step-0 on that road